composer.json

```diff
@@ -6,10 +6,18 @@
 	"require": {
         "php": ">=7.2.0",
         "php-64bit": "*"
-    },
+	},
+	"require-dev": {
+		"phpunit/phpunit": "^6.5"
```

Comment:
We prefer using a phar for PHPUnit. Also, this can be raised to PHPUnit 7 since this library is strictly PHP 7.2 and upwards only.